rke2/ayekan/exporters/snmp-generator/Makefile

```diff
@@ -18,43 +18,28 @@ MIB_PATH := 'mibs'
 
 CURL_OPTS ?= -L -sS --retry 3 --retry-delay 3 --fail -A "Prometheus SNMP generator Makefile"
 
-APC_URL             := https://download.schneider-electric.com/files?p_enDocType=Firmware&p_File_Name=powernet451.mib&p_Doc_Ref=APC_POWERNETMIB_451_EN
+APC_URL             := https://download.schneider-electric.com/files?p_Doc_Ref=APC_POWERNETMIB_EN&p_enDocType=Firmware&p_File_Name=powernet457.mib
 ARISTA_URL          := https://www.arista.com/assets/data/docs/MIBS
-CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/2d465cce2de4e67a3561d8e41e4c99b597558d4b/v2
+CISCO_URL           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v2/
+CISCO_URL2           := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v1/
```

Comment:
```suggestion
CISCO_URL2          := https://raw.githubusercontent.com/cisco/cisco-mibs/05dbf50226f7df5f52fd2dd1a9c17759273fa0d0/v1/
```

Reply:
There's an extra space here.

Why do we need the old cisco MIB?

Reply:
oh I didnt noticed the extra space., good catch.

The old cisco is needed for the CPU mibs. Newer mibs are based on processes and some of our equipment do not support it.

Reply:
oh I didnt noticed the extra space. 

The old cisco is needed for the CPU mibs. Newer mibs are based on processes and some of our equipment do not support it.